src/browser/components/MainPage.jsx

```diff
@@ -365,6 +385,14 @@ const MainPage = createReactClass({
         <Grid fluid={true}>
           { tabsRow }
           { viewsRow }
+          { this.state.finderVisible ? (
+            <Finder
+              webviewKey={this.state.key}
+              close={this.toggleFinder}
```

Comment:
`close` prop still refers toggleFinder(), so the finder doesn't close when clicking "x" button.

Reply:
`toggleFinder ` was right but there is another issue with event propogation. Changed to use explicit function for closing. Thanks for the review